machines/arch_filelist.mk

```diff
@@ -0,0 +1,118 @@
+# This file contains a list of the *core* manycore architecture files that are
+# used in simulation. This has NOT been used in tapeout or any tapeout related
+# activities beyond simulation
+
+ARCH_VINCDIRS =$(BASEJUMP_STL_DIR)/bsg_misc
+ARCH_VINCDIRS+=$(BASEJUMP_STL_DIR)/bsg_cache
+ARCH_VINCDIRS+=$(BASEJUMP_STL_DIR)/bsg_noc
+ARCH_VINCDIRS+=$(BSG_MANYCORE_DIR)/v
+ARCH_VINCDIRS+=$(BSG_MANYCORE_DIR)/v/vanilla_bean
+
+ARCH_VDEFINES =$(BASEJUMP_STL_DIR)/bsg_misc/bsg_defines.v
+ARCH_VDEFINES+=$(BASEJUMP_STL_DIR)/bsg_noc/bsg_noc_pkg.v
+ARCH_VDEFINES+=$(BASEJUMP_STL_DIR)/bsg_cache/bsg_cache_pkg.v
+ARCH_VDEFINES+=$(BASEJUMP_STL_DIR)/bsg_fpu/bsg_fpu_pkg.v
+
+ARCH_VSOURCES =$(BASEJUMP_STL_DIR)/bsg_misc/bsg_less_than.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_reduce.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_abs.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_mul_synth.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_priority_encode.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_priority_encode_one_hot_out.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_encode_one_hot.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_scan.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_mux.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_mux_one_hot.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_mux_segmented.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_dff.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_dff_en_bypass.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_dff_en.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_dff_reset.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_transpose.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_crossbar_o_by_i.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_decode_with_v.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_decode.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_counter_clear_up.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_counter_up_down.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_round_robin_arb.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_circular_ptr.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_imul_iterative.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_idiv_iterative.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_idiv_iterative_controller.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_buf.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_buf_ctrl.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_xnor.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_nor2.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_adder_cin.v
+ARCH_VSOURCES+=$(BASEJUMP_STL_DIR)/bsg_misc/bsg_cycle_counter.v
```

Comment:
bsg_cycle_counter should be in the testbench list

Reply:
I worked on existing filelist. So, cannot pay attention to this detail:
https://github.com/bespoke-silicon-group/bsg_manycore/blob/b4e4cc7625e8318b20ae66006414a8b07703153e/machines/core.include#L41

I'll move it in the next update.